RecoBTag/SoftLepton/python/softLepton_cff.py

```diff
@@ -8,3 +8,32 @@
 from RecoBTag.SoftLepton.SoftLeptonByPt_cff import *
 from RecoBTag.SoftLepton.SoftLeptonByIP3d_cff import *
 from RecoBTag.SoftLepton.SoftLeptonByIP2d_cff import *
+
+softLeptonTask = cms.Task(
+    softPFElectronsTagInfos,
+    softPFMuonsTagInfos,
```

Comment:
how do the 6 imports map to the 26 modules entered in this task?
Can this be made in a more maintainable way?